compiler/ballerina-backend-llvm/src/main/ballerina/src/compiler_backend_llvm/llvm_gen.bal

```diff
@@ -105,13 +111,29 @@ function mapFuncsToNameAndGenrator(llvm:LLVMModuleRef mod, llvm:LLVMBuilderRef b
     map<FuncGenrator> genrators = {};
     foreach var func in funcs {
         if (!(func is ())) {
+            //Skipping init, start, stop function temporarily. TODO: Remove this check
+            if (!checkForValidFunction(func)) {
+                continue;
+            }
             FuncGenrator funcGen = new(func, mod, builder);
             genrators[func.name.value] = funcGen;
         }
     }
     return genrators;
 }
 
+function checkForValidFunction(bir:Function func) returns boolean {
+    if (stringutils:contains(func.name.value, "init")) {
```

Comment:
Shouldn't these be exact matches or atleast match with "__init"?

Reply:
Yes I think they should Me. I'll change these values